common/meta/sled-store/src/sled_tree.rs

```diff
@@ -97,6 +117,53 @@ impl SledTree {
         Ok(got)
     }
 
+    /// Retrieve the value of key.
+    pub fn get<KV: SledKeySpace>(&self, key: &KV::K) -> common_exception::Result<Option<KV::V>> {
+        let got = self
```

Comment:
If it is possible, do not move codes in a PR that adds new feature.

Hard to see if the moved code is changed or not... :(((